infrastructure/terraform/terraform.tfvars

```diff
@@ -0,0 +1,12 @@
+project_id  = "konomi-ai"
+
+# Pick a region with low spot VM prices. us-west4 is currently the cheapest.
+# https://cloud.google.com/compute/vm-instance-pricing
+region           = "us-west4"
+zone             = "us-west4-a"
+gke_cluster_name = "chronicle-staging"
+number_of_nodes  = 1
+machine_type     = "e2-medium"
```

Comment:
I recommend we use `e2-micro` instead, none of our services are that compute heavy. We do need our server to scale automatically based on the workload, is that possible with our current setup?

Reply:
for prod purposes, we could looks at [horizontal pod autoscaling](https://registry.terraform.io/providers/hashicorp/kubernetes/latest/docs/resources/horizontal_pod_autoscaler) although I would rather it be included in the [manifest](https://kubernetes.io/docs/tasks/run-application/horizontal-pod-autoscale-walkthrough/).

Reply:
I'll upgrade the cluster to use `e2-micro` and yup we can use horizontal pod autoscaling to scale based on load in the future